src/DTO/UserDTO.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+class UserDTO {
```

Comment:
perkeliam šią klasę iš aplanko `DTO` į aplanką `Entity` (vienintelė to priežastis - po truputį darom tavo dienyną panašų į projektą naudojantį Symfony framework). Taip pat iš visur pašalinam DTO